ui/components/multichain/connected-accounts-menu/connected-accounts-menu.tsx

```diff
@@ -36,13 +36,15 @@ export const ConnectedAccountsMenu = ({
   disableAccountSwitcher = false,
   onClose,
   closeMenu,
+  onActionClick,
 }: {
   isOpen: boolean;
   identity: Identity;
   anchorElement: HTMLElement | null;
   disableAccountSwitcher: boolean;
   onClose: () => void;
   closeMenu: () => void;
+  onActionClick: (mesg: string) => void;
```

Comment:
Total nit but can this be `message`?  Abbreviations like this aren't necessary.

Reply:
Updated here https://github.com/MetaMask/metamask-extension/pull/23624/commits/19e5b4dd8efbaf825ebe7a0c030f7a89f505cb4e